docs/stencil-docs/customizing-checkout/optimized-one-page-checkout.md

```diff
@@ -2,19 +2,21 @@
 
 <div class="otp" id="no-index">
 
-### On This Page
-- [Working with the Optimized Checkout SCSS File](#working-with-the-optimized-checkout-scss-file)
-- [Configuring the Desktop Viewport](#configuring-the-desktop-viewport)
-- [Classes Available for Customization](#classes-available-for-customization)
-- [Providing Customizable Options in Store Design](#providing-customizable-options-in-store-design)
-- [Currency Conversion Options](#currency-conversion-options)
-- [Resources](#resources)
+### On this page
+- [Working with the optimized checkout SCSS file](#working-with-the-optimized-checkout-scss-file)
+- [Configuring the desktop viewport](#configuring-the-desktop-viewport)
+- [Classes available for customization](#classes-available-for-customization)
+- [Providing customizable options in Page Builder](#providing-customizable-options-in-page-builder)
+
+- [Currency conversion options](#currency-conversion-options)
+- [Related resources](#related-resources)
+
 
 </div> 
 
-## Working with the Optimized Checkout SCSS File
+## Working with the optimized checkout SCSS file
 
-Cornerstone includes an SCSS file that styles the Optimized Checkout page. In your local installation, this file is located at <span class="fn">cornerstone/assets/scss/optimized-checkout.scss</span>.
+Cornerstone includes a SCSS file that styles the Optimized Checkout page. In your local installation, this file is located at <span class="fn">cornerstone/assets/scss/optimized-checkout.scss</span>.

```

Comment:
All of the <span class="fn"></span> instances in the document can be replaced with backticks for consistency.